pvlib/bifacial/infinite_sheds.py

```diff
@@ -276,6 +276,8 @@ def get_irradiance_poa(surface_tilt, surface_azimuth, solar_zenith,
       [W/m^2]
     - ``poa_ground_diffuse`` : total ground-reflected diffuse irradiance on the
       plane of array. [W/m^2]
+    - ``shaded_fraction`` : fraction of the surface, going up from the bottom,
```

Comment:
I think this description (and for the corresponding front/back quantities) should be similar to that used for `f_x` in `_shaded_fraction`:

"Fraction of row slant height from the bottom that is shaded from direct irradiance by adjacent rows."

Reply:
@cwhanse  - just to be clear, you are suggesting I revert back to the way it was, removing the changes that @kandersolar suggested?

I.e., revert back to,

in `get_irradiance_poa`:
```
    - ``shaded_fraction`` : fraction of row slant height from the bottom that
      is shaded from direct irradiance. [unitless]
```

in `get_irradiance`:
```
    - ``shaded_fraction_front`` : fraction of row slant height from the
      bottom that is shaded from direct irradiance on the front surface.
      [unitless]
.
.
.
    - ``shaded_fraction_back`` : fraction of row slant height from the
      bottom that is shaded from direct irradiance on the back surface.
      [unitless]
```

Reply:
That's closer to what @williamhobbs originally had before I tried to improve it :) so long as it mentions something about the shading coming from other rows, I am fine with anything here.